src/sonic-yang-models/yang-models/sonic-crm.yang

```diff
@@ -404,6 +404,46 @@ module sonic-crm {
                     type threshold;
                 }
 
+		leaf srv6_my_sid_entry_threshold_type{
```

Comment:
>		 [](http://example.com/codeflow?start=0&length=2)

Mixing tabs and spaces in the same file. We prefer 4 spaces than 1 tab.

Reply:
fixed